SparkPost/2web2ui
# AC-862: Removed X-Sparky header on authenticate

[AC-862](https://jira.int.messagesystems.com/browse/AC-862)

### What Changed
 - Removed header on `/authenticate`

### How To Test
 - Run multiple requests by navigating pages. Check that `X-Sparky` header is on any request except `/authenticate`

### Notes
Header is already removed API side, but this just makes sure it's gone client side for `/authenticate`. Another alternative is to add the header as an option, and then make it enabled everywhere but `/authenticate` rather than removing it

### To Do
